Core/clim-core/menu.lisp

```diff
@@ -179,7 +187,10 @@ account, and create a list of menu buttons."
       (disown-frame frame-manager submenu-frame)
       (disarm-gadget sub-menu)
       (dispatch-repaint sub-menu +everywhere+)
-      (setf submenu-frame nil) )))
+      (setf submenu-frame nil)))
+  (with-application-frame (frame)
+    (when frame
+      (setf (slot-value frame 'active-menu) nil))))
```

Comment:
make active-menu an accessor on frame (here you use setf on slot-value and earlier you use with-slots - this is not consistent).

Reply:
In the other place I accessed the slot twice. Here only once. I try to always use with-accessors/with-slots in any place where there are multiple uses, and avoid it when there is only one.
I changed this to use the accessor in this case.